public/docs/_examples/cb-item-template/ts/app/main.ts

```diff
@@ -0,0 +1,13 @@
+/* tslint:disable */
+// #docregion
+import { bootstrap } from '@angular/platform-browser-dynamic';
+
+import { AppComponent } from './app.component';
+
+bootstrap(AppComponent).then(
+  () => window.console.info('Angular finished bootstrapping your application!'),
+  (error) => {
+    console.warn('Angular was not able to bootstrap your application.');
```

Comment:
Are we using `console.warn` in other examples as well? If not I would get rid of it.